javascript1/week4/lesson-plan.md

```diff
@@ -1,65 +1,35 @@
 # Lesson plan
 ```
```

Comment:
Is there a reason this is formatted as code? It could be normal text, taking advantage of lists, better format, etc